libs/features/mas/test/hydrate.test.js

```diff
@@ -298,8 +298,8 @@ describe('processBackgroundImage', () => {
     });
 
     it('should not process background image when fields.backgroundImage is falsy', () => {
-        const fields = { backgroundImage: null };
-        const backgroundImageConfig = { tag: 'div', slot: 'background' };
+        const fields = { backgroundImage: null, backgroundImageAltText: 'Test Image' };
+        const backgroundImageConfig = { tag: 'div', slot: 'image' };
```

Comment:
it seems you are missing the `image` slot in the `ccd-suggested` layout?

Reply:
In this case `ccd-suggested` uses `background-image` instead of `image` (the image is used as a css `background-image` property).

Reply:
Looking at https://github.com/adobecom/milo/blob/8f4ef3b66e15373294adb4303f935387524089ee/libs/features/mas/src/variants/ccd-suggested.js

I don't see neither of a slot for the background image or use of backgroundImage.
What am I missing?